src/components/Common/SingleService.js

```diff
@@ -0,0 +1,16 @@
+import React, {Component} from 'react';
+class SingleService extends Component{
+    render(){
+        return (
+            <div className="col-md-4">
+                <span className="fa-stack fa-4x">
+                    <i className="fa fa-circle fa-stack-2x text-primary"></i>
+                    <i className={`fa ${this.props.icon} fa-stack-1x fa-inverse`}></i>
+                </span>
+                <h4 className="service-heading">{this.props.title}</h4>
```

Comment:
You could extract the variables you're using here, just before the render, voiding the constantly use of `this.state`
At line 3 you can do:
```
const { title, description } = this.props;
```

And then here just do:
```
<h4 className="service-heading">{title}</h4>
```